### PR TITLE
Feat/new-links-1858: added additional links to external resources

### PR DESCRIPTION
**Pull request by amandadupell**
#### What does this PR do?

Adds more resources to the Accessibility page that were in part taken from the Product Accessibility Office and in part from links I found to be most helpful.

#### Where should the reviewer start?

Accessibility page

#### What testing has been done on this PR?

Manual

#### How should this be manually tested?

Visiting Accessibility page

#### What are the relevant issues?

#1858 

#### Screenshots (if appropriate)

Screen Shot 2021-09-21 at 5 47 08 PM

#### Should this PR be mentioned in Design System updates?

We could highlight that new resources and a checklist has been added to the Accessibility page

#### Is this change backwards compatible or is it a breaking change?

Backwards compatible

### CODE REVIEW

**Review by taysea — APPROVED** [review]
LGTM, I like separating the internal vs external resources

File: aries-site/src/pages/foundation/accessibility.mdx [hunk 1]
```diff
@@ -50,13 +50,17 @@ should still be performed, Grommet handles many of the general DOM requirements
...
+## Internal Resources
 
 - [HPE Video Guides on Accessibility (HPE internal link)](https://hpe.sharepoint.com/sites/f5/cto/office/Accessibility/Pages/accessible-design-training.aspx): A collection of educational videos from HPE Product Accessibility Program Office to assist all stakeholders in understanding and implementing accessible design.
 - [HPE Accessible Web, Software & Hardware Design Guides (HPE internal link)](https://hpe.sharepoint.com/sites/f5/cto/office/Accessibility/Pages/accessible-web-software-and-hardware-design.aspx): Text-based guides from HPE Product Accessibility Program Office on designing and implementing accessible products.
+
+## External Resources
+
 - [WCAG 2.1](https://www.w3.org/TR/WCAG21/): A robust set of accessibility design guidelines for software and web-based products. HPE policy is to conform to all applicable guidelines identified as Level A and AA.
+- [WebAIM Introduction to Web Accessibility](https://webaim.org/intro/): An all-encompassing introducation to how accessibility can improve the web for all users.
```
Comment (halocline) [inline aries-site/src/pages/foundation/accessibility.mdx:61]:
```suggestion
- [WebAIM Introduction to Web Accessibility](https://webaim.org/intro/): An introduction on how accessibility improves the Web for all users.
```

File: aries-site/src/pages/foundation/accessibility.mdx [hunk 2]
```diff
@@ -50,13 +50,17 @@ should still be performed, Grommet handles many of the general DOM requirements
...
+- [The A11y Project Checklist](https://www.a11yproject.com/checklist/): A checklist that uses WCAG 2.1 as a reference for developers and designers to ensure their product's accessibility.
```
Comment (halocline) [inline aries-site/src/pages/foundation/accessibility.mdx:62]:
```suggestion
- [The A11y Project Checklist](https://www.a11yproject.com/checklist/): A checklist using WCAG 2.1 as a reference for developers and designers to ensure their product's accessibility. 
```

**Review by halocline — COMMENTED** [review]
Suggestions for using a more "active" voice + typo fix.

**Comment by amandadupell** [comment]
> Suggestions for using a more "active" voice + typo fix.

thanks for the edits! sounds better

**Review by halocline — APPROVED** [review]

**Pull request merged**